1-Authentication/0-sign-in-vanillajs/App/sample.test.js

```diff
@@ -1,68 +1,58 @@
-/**
- * @jest-environment jsdom
- */
-
-const request = require('supertest');
-const path = require('path');
-const fs = require('fs');
-
-const app = require('./server.js');
-
-jest.dontMock('fs');
-
-const html = fs.readFileSync(path.resolve(__dirname, './App/index.html'), 'utf8');
-
-describe('Sanitize index page', () => {
-    beforeAll(async() => {
-        global.document.documentElement.innerHTML = html.toString();
-    });
-
-    it('should have valid cdn link', () => {
```

Comment:
Tutorial doesn't actually use the CDN link option, so removed this test.

Also updated some file paths so that the tests pass.